src/types/create_item.rs

```diff
@@ -0,0 +1,96 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use serde::Deserialize;
+use xml_struct::XmlSerialize;
+
+use crate::{
+    types::sealed::EnvelopeBodyContents, BaseFolderId, Items, Operation, OperationResponse,
+    ResponseClass, ResponseCode, MESSAGES_NS_URI,
+};
+
+/// Describes how an item is handled once it has been created, if it's a message
+/// item.
+///
+/// See <https://learn.microsoft.com/en-us/exchange/client-developer/web-service-reference/createitem#messagedisposition-attribute>
+#[derive(Debug, XmlSerialize)]
+#[xml_struct(text)]
+pub enum MessageDisposition {
+    SaveOnly,
+    SendOnly,
+    SendAndSaveCopy,
+}
+
+/// A request to create (and optionally send) one or more Exchange item(s).
+///
+/// See <https://learn.microsoft.com/en-us/exchange/client-developer/web-service-reference/createitem>
+#[derive(Debug, XmlSerialize)]
+#[xml_struct(default_ns = MESSAGES_NS_URI)]
+pub struct CreateItem {
+    /// Describes how an item is handled once it has been created, if it's a
+    /// message item.
+    ///
+    /// See <https://learn.microsoft.com/en-us/exchange/client-developer/web-service-reference/createitem#messagedisposition-attribute>
+    #[xml_struct(attribute)]
+    pub message_disposition: MessageDisposition,
+
+    /// The folder in which to store an item once it's been created, if it's a
+    /// message item.
```

Comment:
todo: EWS docs suggest this isn't only applicable to `Message`s.

nitpick: I generally prefer to avoid contractions in technical writing.

```suggestion
    /// The folder in which to store an item once it has been created.
```